data/language/dutch.txt

```diff
@@ -3368,9 +3368,9 @@ STR_3366    :{BLACK}= Attractie
 STR_3367    :{BLACK}= Eetkraampje
 STR_3368    :{BLACK}= Drankkraampje
 STR_3369    :{BLACK}= Souvenirkraampje
-STR_3370    :{BLACK}= Infokiosk
+STR_3370    :{BLACK}= Informatiekiosk
```

Comment:
Dat begrijp ik, maar 'Informatiekiosk' voluit is nog steeds korter dan 'Souvenirkraampje'. Derhalve denk ik dat het geen probleem moet opleveren.